### DISCUSSION TITLE
⛔ 🛟 Support Request: Custom CSS

### DISCUSSION THREAD

**Discussion by Vadivel-Nallasamy** [elementor/elementor#21197] [Developers Community]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The custom css code used to change the color of specific characters in a word in animated text widget doesn't works. Find out the bug and please enable the editing.

### Describe the solution you'd like

I don't have any solution for this.

**Comment by loucasl** [comment]
What is the CSS you're trying to use?

**Reply by Vadivel-Nallasamy** [reply]
I was looking to change the color of specific letters in rotating text of animated text widget using `<span style = "color: red">Text<\span>` but it was not working there.

**Reply by m1ga** [reply]
![Screenshot_20230131_211414](https://user-images.githubusercontent.com/4334997/215872359-e10de06b-a02a-4b00-8c8d-3bac841e11e1.png)

**Reply by m1ga** [reply]
or just `<span>Ama</span>zing` and custom css
![Screenshot_20230131_211751](https://user-images.githubusercontent.com/4334997/215873252-8ecd1be4-56ee-482e-9dc3-439d5980cda1.png)

multiple options

**Reply by Vadivel-Nallasamy** [reply]
Sir, It works with highlighted style. I want it to work with rotating style (Animation: Typing).

**Reply by m1ga** [reply]
Ah ok. I thought you were talking about the text in the rotating animation (the circle). Your original question doesn't have those details

**Reply by Vadivel-Nallasamy** [reply]
Alright sir! Is there a way to make it work? If yes, please provide me the solution and answer to "Why does the same function is not working with rotating text style but with highlighted text style?".

**Reply by m1ga** [reply]
because it will break up your word into each character automatically:
![2023-01-31_22-13](https://user-images.githubusercontent.com/4334997/215884566-7c9ababa-c136-4af3-96a4-0e09e8bf0989.png)
So you have a span around each letter already with a class. So use my 2nd approach with the custom css and apply it to the letter that you want to change. Should be doable with basic CSS selectors. I'm not on my work computer so I can't access my Elementor page. This is from the demo frontend

**Reply by m1ga** [reply]
and for the "Rotating" option I guess it is just taking your word and adds a span around (it takes each line and makes a span, so it looks like its removing custom HTML from the field) it so you'll have to use custom JS to split it

**Reply by Vadivel-Nallasamy** [reply]
And it's confusing as hell. Try to find a solution when you get free time sir. Thank you.

**Reply by m1ga** [reply]
like I've said: use the Custom CSS with the .elementor-headline-dynamic-letter classes and CSS selectors to access each character

**Comment by nicholaszein** [comment]
Hello @Vadivel-Nallasamy ,

Thanks for contacting us! 🙏 

### 🛟 This is more of a support question, not really something we can help with here. 

👉 Please remember to check the [Help Center](https://elementor.com/help) or contact [Elementor Support](https://elementor.com/contact) before submitting an issue.


### ⚠️ Note that GitHub Discussions are reserved for Elementor 🚀 Feature Requests only, and should not be used for support.

❔ If you have questions, or need guidance, help, or general support, please use one of the following channels:

- [Elementor Global Community](https://facebook.com/groups/Elementors/) The main Facebook group where all kinds of users come together to help each other.
- [Elementor Academy](https://elementor.com/academy) Learn everything you need to know about using Elementor.
- [Elementor Developers Documentation](https://developers.elementor.com/) Learn everything about our Developer-API.
- [Free Support Forum](https://wordpress.org/support/plugin/elementor) Get help on WordPress.org.

💎 If you have an active [Elementor Pro](https://elementor.com/pro) license you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.

### ⚠️ As a gentle reminder, **🛟 Support Request issues** may need to be closed without troubleshooting. 

Thank you for understanding! 🙏 

Kind regards
